test/trip_updates/supervisor.exs

```diff
@@ -0,0 +1,19 @@
+defmodule TripUpdates.SupervisorTest do
```

Comment:
suggestion: There's already a [supervisor_test.exs](https://github.com/mbta/alerts_viewer/pull/75/files#diff-666eeee3a1c72a9d85222564740be9f9751c915a2ded8f85ef1308e2487bacd8) file here, I think this is now just a duplicate and can be removed? (This one is also missing the `_test` in the filename, FWIW.)